presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -255,10 +262,12 @@ public Ssl()
     {
         private static final Predicate<Properties> IF_SSL_ENABLED =
                 checkedPredicate(properties -> SSL.getValue(properties).orElse(false));
+        private static final Predicate<Properties> IF_SSL_VERIFICATION_ENABLED =
+                checkedPredicate(properties -> !SSL_VERIFICATION.getValue(properties).orElse(SslVerificationMode.FULL).equals(SslVerificationMode.NONE));
```

Comment:
Can we set it as a default value in `AbstractConnectionProperty` ?

Reply:
Thank you for the review.

Currently, `SSLVerification` is only allowed if `SSL` is enabled. If a default value is set to `SSLVerification`, when `SSL` is not enabled (default), the following exception will be thrown: 
`java.sql.SQLException: Connection property 'SSLVerification' is not allowed`

In order to resolve this problem, should `SSLVerification` be allowed regardless of whether `SSL` is enabled or not?